modules/statics/test/unit/coins.ts

```diff
@@ -61,4 +61,78 @@ describe('All Coins', () => {
     dot.decimalPlaces.should.equal(12);
     dot.primaryKeyCurve.should.equal('ed25519');
   });
+
+  describe('ARC20 and AVAXERC20 Tokens', () => {
+    it('Token PNG', () => {
+      const png = coins.get('PNG');
+      png.fullName.should.equal('Pangolin');
+      png.name.should.equal('PNG');
+      png.isToken.should.true;
+      png.asset.should.equal('png');
+      png.decimalPlaces.should.equal(18);
+      png.network.name.should.equal('AvalancheCTestnet');
+      png.network.type.should.equal('testnet');
+      png.network.family.should.equal('avaxc');
+    });
+
+    it('Token Qi', () => {
+      const qi = coins.get('Qi');
```

Comment:
Should the 'i' be in uppercase in the token name?

Reply:
Done